cspell.json

```
@@ -438,6 +438,7 @@
         "numberformat",
         "objc",
         "oblador",
+        "OCBC"
```

You need a comma here. I think that should fix the errors @twisterdotcom 👍 

```suggestion
        "OCBC",
```